src/index.ts

```diff
@@ -16,6 +17,15 @@ const client = new Client({ partials: ['MESSAGE', 'CHANNEL', 'REACTION'] });
     // rig up client callbacks
     client.on('error', console.error);
 
+    Mongoose.connect(
+      config.MONGO_URI,
+      {
+        useNewUrlParser: true,
+        useUnifiedTopology: true
+      },
+      () => console.log('MongoDB ready!')
+    );
+
```

Comment:
Add a check/flag around this to make sure MONGO_URI is passed, also add an `await` for the `.connect` call

Reply:
Stuck it in an `if(config.MONGO_URI)` block.